OracleDatabase/samples/prebuiltdb/README.md

```diff
@@ -0,0 +1,101 @@
+# Example of creating an image with pre-built DB
+=================================================
+Warning! The description below requires changes in the dockerfile related to the version and edition in question (docker-images/OracleDatabase/dockerfiles/$VERSION/Dockerfile.$EDITION). It's recommended that you revert the changes after you have completed creating image without VOLUME (see next section).
+
+## 1. Create an image without a VOLUME
+
+When creating an image with a pre-build DB, we first need an image without a VOLUME. To create such an image you'll need to comment out the VOLUME command in the dockerfile in use (docker-images/OracleDatabase/dockerfiles/$VERSION/Dockerfile.$EDITION).
+
+Comment out the following line:
+```
+#VOLUME ["$ORACLE_BASE/oradata"]
+```
+In the example below I have edited the Dockerfile for version 12.1.0.2 and Enterprise Edition (ee) (docker-images/OracleDatabase/dockerfiles/12.1.0.2/Dockerfile.ee).
+
+If you want to keep your existing image, you need to rename it:
+```
+docker tag oracle/database:12.1.0.2-ee oracle/database-org:12.1.0.2-ee
+```
+
+Then create an image without a VOLUME:
+```
+cd docker-images/OracleDatabase/dockerfiles
+sh buildDockerimage.sh -v 12.1.0.2 -e -i 
```

Comment:
Typo, image is spelled with a capital "I": `sh buildDockerImage.sh -v 12.1.0.2 -e -i`